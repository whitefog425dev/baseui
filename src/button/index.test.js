// @flow

import React from 'react';
import Enzyme, {shallow} from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';

import Button from './index';

Enzyme.configure({adapter: new Adapter()});

test('<Button />', () => {
  const wrapper = shallow(<Button onClick={() => {}} />);
  expect(wrapper.text()).toBe('it is a button!');
});
