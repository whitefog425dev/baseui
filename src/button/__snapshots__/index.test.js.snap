// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Button /> 1`] = `
<button
  onClick={[Function]}
  type="button"
>
  it is a button!
</button>
`;
